imod/tests/test_mf6/test_multimodel/test_mf6_partitioning_unstructured.py

```diff
@@ -206,3 +206,63 @@ def test_partitioning_unstructured_with_vpt_cells(
     np.testing.assert_allclose(
         head["head"].values, orig_head.values, rtol=1e-5, atol=1e-3
     )
+
+
+@pytest.mark.usefixtures("circle_model")
+@pytest.mark.parametrize(
+    "partition_name",
+    ["two_parts", "three_parts"],
+)
+def test_partitioning_unstructured_with_well(
+    tmp_path: Path, circle_model: Modflow6Simulation, partition_name: str
+):
+    simulation = circle_model
+    # increase the recharge to make the head gradient more pronounced
+    simulation["GWF_1"]["rch"]["rate"] *= 100
+
+    # Add well
+    well = imod.mf6.Well(
```

Comment:
why do you add just 1 well? Isn't that a bit meager for testing this functionality?

Reply:
Good point, added this. Immediately found a bug and fixed it! https://github.com/Deltares/imod-python/issues/700